app/src/vanilla/java/cm/aptoide/pt/app/view/MoreBundleView.java

```diff
@@ -0,0 +1,49 @@
+package cm.aptoide.pt.app.view;
+
+import cm.aptoide.pt.home.AdClick;
+import cm.aptoide.pt.home.AppHomeEvent;
+import cm.aptoide.pt.home.HomeBundle;
+import cm.aptoide.pt.home.HomeEvent;
+import cm.aptoide.pt.presenter.View;
+import java.util.List;
+import rx.Observable;
+
+/**
+ * Created by D01 on 05/06/2018.
+ */
+
+public interface MoreBundleView extends View {
```

Comment:
This interface already exists (homeview) we should probably just add the toolbarinfo method and extend the homeview (but maybe change the "homeview" to bundles view or something)